lib/datasource/index.ts

```diff
@@ -111,6 +114,47 @@ export async function getPkgReleases(config: PkgReleaseConfig) {
   }
   return res;
 }
+function getRawReleases(config: PkgReleaseConfig): Promise<ReleaseResult> {
```

Comment:
@vladbivanov You have duplicated functions `fetchReleases` and `getRawReleases` now, please cleanup

Reply:
fixed, web-based merge on github needs a bit more work when compared to gitlab IMO.